fastlane/Fastfile

```diff
@@ -0,0 +1,69 @@
+# This file contains the fastlane.tools configuration
```

Comment:
Ruby code style is a bit inconsistent here. Our `.editorconfig` file should cover it. Maybe you're missing the Android Studio plugin for editorconfig?